rpc/params/charms.go

```diff
@@ -200,6 +199,8 @@ type CharmBase struct {
 type CharmContainer struct {
 	Resource string       `json:"resource,omitempty"`
 	Mounts   []CharmMount `json:"mounts,omitempty"`
+	Uid      int          `json:"uid,omitempty"`
```

Comment:
Do we have compatibility concerns here? I guess not because this struct is used as a result of an api call so will always be received by a worker in sync with the controller.

Reply:
I'll have to check that, but I believe clients just ignore fields they don't care about when unpacking the charm metadata anyway.